src/main/java/frc/robot/commands/autonomous/FiveCycle.java

```diff
@@ -16,14 +16,19 @@
 import frc.robot.RobotContainer;
 import frc.robot.subsystems.arm.ArmConstants;
 import frc.robot.subsystems.elevator.ElevatorConstants;
+import frc.robot.subsystems.groundIntake.GroundIntakeConstants;
+import frc.robot.subsystems.groundIntake.GroundIntakeSubsystem;
 
 public class FiveCycle extends PathPlannerAuto {
 
   private RobotContainer robotContainer;
 
   private double waitTime = 0;
 
-  public FiveCycle(RobotContainer robotContainer, boolean nonProcessorSide) {
+  public FiveCycle(
```

Comment:
This is me being nitpicky honestly, but it would be better if we swapped the arguments around. That way, the `nonProcessorSide` argument is at the end, which makes it easier to read and follow

```suggestion
  public FiveCycle(
    RobotContainer robotContainer,
    GroundIntakeSubsystem groundIntakeSystem,
    boolean nonProcessorSide) {
```

Reply:
So the pattern in general for method parameters:
robotContainer, subsystem1, subsystem2, ..., other parameters?